### ISSUE TITLE
public forum

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#3546]
Story:
- when discussing a pain point with my API team, i want to open up the discussion to more than just my team. maybe i want to open it up to the whole engineering department. Maybe the whole organization. maybe the whole world.
- when i open up a topic to the whole world, I want 
  - to read a primer to know what the best in class practices are for others
  - to discuss with people in different organizations
  - to trust that the discussion is with other experts

To solve for these issues, I think we need to do the following:
- publish articles on popular pain points. these should be living documents that change as better thinking comes along.
- provide suggested governance templates that can be forked
- provide a public forum where people can discuss certain topics
- make the forum read-only to the whole world (good SEO) but only PRO users can contribute (cheap & easy moderation)   

The articles & forum bring in everybody. 
The governance templates is easy to understand, but difficult to copy & paste (e.g. infographic).  There's an "add to team" button that adds it to your team on parabol, which encourages sign ups. Then, contributing to the discussion is locked to PRO users. 

Basically it's a 3 stage funnel:
1. bring em in with good content. 
2. make em sign up with an interesting new strategy to try. 
3. get em to go pro by providing them a community.

**Comment by jordanh** [comment]
Scrubbed iceboxing for now
